src/components/deviceMap/deviceMap.js

```diff
@@ -167,6 +167,18 @@ class DeviceMap extends Component {
     });
   }
 
+  renderStaticMap() {
+      return <div className="map-container">
+        <div className="static-map-modal">
+          <span className="static-text">
+            {lang.MAP_MODAL_CONTENT}
+            <a className="static-link" rel="noopener noreferrer" href="https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-faq" target='_blank'>{lang.MAP_MODAL_FAQS}</a>
```

Comment:
please remove the "en-us" from the URL

Reply:
Just for making sure. In comps they have given me this " https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-faq". Even if i eliminate the "en-us" it s still redirecting to same link with en-us appending. Is that the actula behaviour??

Reply:
I am on it.